Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -111,6 +128,10 @@ open class ChatChannelListVC: _ViewController,
         
         userAvatarView.controller = controller.client.currentUserController()
         userAvatarView.addTarget(self, action: #selector(didTapOnCurrentUserAvatar), for: .touchUpInside)
+        
+        channelListErrorView.refreshButtonAction = { [weak self] in
+            self?.controller.synchronize()
+        }
```

Comment:
Because a customer might want to replace the error and loading view with something that they already have in their app, we should expose the error and loading views with protocols instead of the concrete class. Something like this:
```swift
protocol ChannelListErrorView: UIView {
    var refreshButtonAction: (() -> Void)? { get set }
    etc...
}
```

Reply:
As discussed earlier today, we are going to move forward with the concrete class/inheritance approach since is consistent with the rest of our components but I added documentation on how to customize either of the classes so implementers can easily customize them 👍🏻